specs/merge/fork-choice.md

```diff
@@ -43,25 +43,13 @@ The body of this function is implementation dependent.
 The Consensus API may be used to implement this with an external execution engine.
 
 ```python
-def set_head(self: ExecutionEngine, block_hash: Hash32) -> bool:
-    """
-    Returns True if the ``block_hash`` was successfully set as head of the execution payload chain.
-    """
-    ...
-```
-
-#### `finalize_block`
+HEAD_TYPE_FINALIZED_HEAD = uint64(0)
+HEAD_TYPE_SAFE_HEAD = uint64(1)
```

Comment:
From a consensus standpoint there is just one head being set. I would instead reserve the "safe head" strictly for the API and concerns around that

Reply:
The plan is to add `confirmed_block_hash` to the parameter set of the `engine_forkchoiceUpdated` method to pass this information to the execution engine. Respective change to the [`notify_forkchoice_updated`](https://github.com/ethereum/consensus-specs/blob/dev/specs/merge/fork-choice.md#notify_forkchoice_updated) function should be made by that time